provisioning/roles/kuma/files/apache2/all-servers.conf

```diff
@@ -3,15 +3,6 @@
 # Our servers symlink to this file. Do not move it without first coordinating
 # with WebOps.
 
-<Location /static/fonts>
-    # Only enable CORS for fonts when the request is coming from an approved domain
-    Header unset Access-Control-Allow-Origin
-    SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0
-    SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
-    SetEnvIf Origin "https?://(kuma\.readthedocs\.org)" CORS=$0
-    Header set Access-Control-Allow-Origin %{CORS}e env=CORS
```

Comment:
@groovecoder Do I remember correctly that we decided not to worry about restricting font CORS for approved any more? I'm pretty sure we did, but somehow these lines stuck around for some reason.